cmd/thv-operator/pkg/registryapi/config/config.go

```diff
@@ -247,11 +247,16 @@ func validateRegistryNames(registries []mcpv1alpha1.MCPRegistryConfig) error {
 }
 
 func buildFilePath(registryName string) *FileConfig {
+	return buildFilePathWithFilename(registryName, RegistryJSONFileName)
+}
+
+func buildFilePathWithFilename(registryName, filename string) *FileConfig {
 	return &FileConfig{
-		Path: filepath.Join(RegistryJSONFilePath, registryName, RegistryJSONFileName),
+		Path: filepath.Join(RegistryJSONFilePath, registryName, filename),
 	}
 }
```

Comment:
For https://github.com/stacklok/toolhive/issues/2749 we can probably simplify this to have just this second function with two args and have configmap use that as well (`RegistryJSONFileName` is just a default/fallback, not something we hardcode)